### PR TITLE
feat: Improve outputs for standalone example

### LINKED ISSUE #272: Improve outputs of standalone example
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave _+1_ or _me too_ comments; they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If the issue is assigned to a user, that user is claiming responsibility for the issue.

### Description

The currently the standalone examples only give as output:
 - Non Confidential BigQuery table
 - Confidential BigQuery table

Maybe we could improve by adding more outputs, thinking about what each of the personas might need in their daily activities.

We also could add in the READMEs a section to help the user to know how he could use those outputs.

For example:

 - Security Analyst might take the **bucket name** to look through audit logs

### References

* #264

### PR DESCRIPTION
**Pull request by andersonj-cit**
Fixes #272 

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: examples/standalone/outputs.tf [hunk 1]
```diff
@@ -33,3 +33,153 @@ output "bigquery_non_confidential_table" {
...
+  description = "The name of the bucket created for dataflow in the confidential data pipeline."
+  value       = module.secured_data_warehouse.confidential_data_dataflow_bucket_name
+}
+
+output "data_ingestion_bucket_name" {
+  description = "The name of the bucket created for the data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_dataflow_bucket_name
+}
+
+output "data_ingestion_dataflow_bucket_name" {
```
Comment (erlanderlo) [inline examples/standalone/outputs.tf:82]: is this a duplicate of the above?

File: examples/standalone/outputs.tf [hunk 2]
```diff
@@ -33,3 +33,153 @@ output "bigquery_non_confidential_table" {
...
+  description = "The name of the bucket created for dataflow in the data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_dataflow_bucket_name
+}
+
+output "confidential_data_project_id" {
+  description = "The Project where the confidential datasets and tables are created."
```
Comment (erlanderlo) [inline examples/standalone/outputs.tf:88]: no need to cap "project"

Comment (erlanderlo) [reply examples/standalone/outputs.tf:88]: might make the description consistent with the other project id's unless it's generally consistent with all the other examples that have this in their output.tf file.

File: examples/standalone/README.md [hunk 3]
```diff
@@ -260,6 +259,55 @@ iam.googleapis.com \
...
+- Blueprint type
+- Data ingestion bucket name
+- Data ingestion dataflow bucket name
+- Confidential data project id
+- Data governance project id
+- Data ingestion project id
+- Non-confidential data project id
+- Service account: dataflow controller service account
+
+These outputs can be interesting for data analyst group:
```
Comment (erlanderlo) [inline examples/standalone/README.md:275]: the data analyst if likely interested in the output (i.e the BQ datasets) and data lineage things too rathan than only the input.  The input is usually plumbed by the data engineer.

File: examples/standalone/README.md [hunk 4]
```diff
@@ -260,6 +259,55 @@ iam.googleapis.com \
...
+
+- Data ingestion topic name
+- Pubsub writer service account email
+- Storage writer service account email
+
+These outputs can be interesting for data engineer group:
```
Comment (erlanderlo) [inline examples/standalone/README.md:281]: this persona is interested in the pipeline and anything associated with it such as things from the harness (build jobs, flex templates) and responsible for plumbing the input to the output.

**Review by erlanderlo — APPROVED** [review]
Thanks! lgtm with comments

File: examples/standalone/outputs.tf (hunk 1 above)
Comment (andersonj-cit) [reply examples/standalone/outputs.tf:82]: nice catch

**Review by erlanderlo — APPROVED** [review]
LGTM

**Comment by erlanderlo** [comment]
@andersonj-cit  please select a PR title that confirms to: https://www.conventionalcommits.org/en/v1.0.0/
so that the release will bump the appropriate semver.  I'm guessing it's a `feat`

Once that's selected, we can merge

**Pull request merged**